netbsd/Platform.c

```diff
@@ -323,7 +330,100 @@ bool Platform_getNetworkIO(NetworkIOData* data) {
 }
 
 void Platform_getBattery(double* percent, ACPresence* isOnAC) {
-   // TODO
-   (void)percent;
-   (void)isOnAC;
+   int fd;
+   bool isACAD, isBattery;
```

Comment:
What does the ACAD stand for? Did you mean `isACAdapter`?

Reply:
"AC Adapter". The abbreviation is used in envstat.

Reply:
Kinda non-obvious on first sight though. Took me the actual reference string a few lines down to figure it out. Maybe `isACAdapter` is a better choice?